python/tchannel/tornado/dispatch.py

```diff
@@ -88,13 +79,13 @@ def handle_call(self, request, connection):
                 ErrorCode.bad_request,
                 "Endpoint '%s' for service '%s' is not defined" % (
                     request.endpoint, request.service),
-                request.id,
-            )
+                request.id)
```

Comment:
Styling nits:

If the call is in the form

    foo(
        a,
        b,
        c,
    )

It's okay to leave the `)` on a separate line. If the call is in the form,

    foo(a,
        b,
        c)

Then, you want the `)` on the same line as the last parameter.